tests/src/astro/electromagnetism/unitTestYarkovskyAcceleration.cpp

```diff
@@ -0,0 +1,207 @@
+/*    Copyright (c) 2010-2019, Delft University of Technology
+ *    All rights reserved
+ *
+ *    This file is part of the Tudat. Redistribution and use in source and
+ *    binary forms, with or without modification, are permitted exclusively
+ *    under the terms of the Modified BSD license. You should have received
+ *    a copy of the license with this file. If not, please or visit:
+ *    http://tudat.tudelft.nl/LICENSE.
+ */
+
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MAIN
+
+#include <boost/test/unit_test.hpp>
+#include <Eigen/Core>
+
+#include "tudat/basics/testMacros.h"
+#include "tudat/astro/electromagnetism/yarkovskyAcceleration.h"
+#include "tudat/astro/basic_astro/physicalConstants.h"
+
+#include "tudat/math/basic/linearAlgebra.h"
+#include "tudat/astro/basic_astro/orbitalElementConversions.h"
+#include "tudat/interface/spice/spiceInterface.h"
+#include "tudat/math/integrators/rungeKuttaCoefficients.h"
+#include "tudat/astro/basic_astro/accelerationModel.h"
+#include "tudat/simulation/propagation_setup/dynamicsSimulator.h"
+#include "tudat/simulation/environment_setup/defaultBodies.h"
+#include "tudat/simulation/environment_setup/createBodies.h"
+#include "tudat/math/integrators/createNumericalIntegrator.h"
+
+
+namespace tudat
+{
+
+namespace unit_tests
+{
+
+BOOST_AUTO_TEST_SUITE( test_yarkovsky_acceleration )
+
+const double AU = physical_constants::ASTRONOMICAL_UNIT;
+const double JD = physical_constants::JULIAN_DAY;
+const double yarkovskyParameter = -2.899e-14 * AU / ( JD * JD );
```

Comment:
It looks like this value comes from a specific reference, could you add this to the comments?

Reply:
Indeed, it is from the Hernandez paper for Apophis. I will add a comment